src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -111,6 +138,22 @@ class PanelMediaBrowser extends LitElement {
     });
   }
 
+  private _mediaBrowsed(ev) {
+    if (ev.detail.back) {
+      history.back();
+      return;
+    }
+    let path = "";
+    for (const item of ev.detail.ids.slice(1)) {
+      path +=
+        "/" +
+        encodeURIComponent(
+          `${item.media_content_type},${item.media_content_id}`
+        );
+    }
+    navigate(`/media-browser/${this._entityId}${path}`);
```

Comment:
Google Chrome allows the maximum length of the URL to be of the size 2MB(2048 characters)

We may have to worry about this eventually... I was getting an average of about 80 characters per navigation down into a folder. Meaning 25 times I would be able to drill down. That is a lot of diving but still may be something to think about